application/static/swagger.yml

```diff
@@ -45,18 +48,31 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/Item_create'
+              $ref: '#/components/schemas/ItemRequest'
+            examples:
+              ItemRequest:
+                $ref: '#/components/examples/ItemRequest'
       responses:
         '200':
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Item'
+              examples:
+                Item:
+                  $ref: '#/components/examples/Item'
         '400':
           description: Bad Request
-  /items/<id>:
+  /items/{item_id}:
     get:
       summary: Returns a selected item.
+      parameters:
```

Comment:
Add in parameters so users can properly test PUT, DELETE, GET individual endpoints.